core-bundle/src/Resources/contao/config/config.php

```diff
@@ -380,10 +380,12 @@
 		'purgeTempFolder' => array('Contao\Automator', 'purgeTempFolder'),
 		'purgeSearchCache' => array('Contao\Automator', 'purgeSearchCache'),
 		'generateSitemap' => array('Contao\Automator', 'generateSitemap'),
+	),
+	'hourly' => array
+	(
 		'purgeRegistrations' => array('Contao\Automator', 'purgeRegistrations'),
-		'purgeOptInTokens' => array('Contao\Automator', 'purgeOptInTokens')
+		'purgeOptInTokens' => array('Contao\Automator', 'purgeOptInTokens'),
```

Comment:
This could be a (future-proof) listener as well actually.

Reply:
Should we put all purge cron tasks into the existing `CronJob` may be?